grafana_client/elements/datasource.py

```diff
@@ -95,3 +95,31 @@ def delete_datasource_by_name(self, datasource_name):
         delete_datasource = "/datasources/name/%s" % datasource_name
         r = self.client.DELETE(delete_datasource)
         return r
+
+    def get_datasource_proxy_data(self, datasource_id
```

Comment:
> If there are problems with this PR, just tell me. Particularly concerning code coverage, because I'm really not famillar with the concept.

I can't see where `get_datasource_proxy_data()` will be called from any code, including test cases. This is probably why Codecov reported about a 0.52% decline in code coverage at https://github.com/panodata/grafana-client/pull/5#issuecomment-1030836893.

In order to learn something about the general concept, I would like to refer you to [^1]. Maybe you can add a corresponding test case for this function? Please let me know if you need further assistance or clarification on this detail.

[^1]: https://en.wikipedia.org/wiki/Code_coverage

Reply:
I see that the documentation section [^1] does not contain any example about how to use the `datasource` element. Feel free to add a short example there about its baseline usage, and in particular about its usage with `get_datasource_proxy_data`. Do you think this is a good idea?

[^1]: https://github.com/panodata/grafana-client#getting-started